src/components/Header.tsx

```diff
@@ -18,7 +18,7 @@ export const HEADER_HEIGHT = Platform.select({
 }) as number;
 
 interface Props {
-  navigation?: StackNavigationProp<any, any>;
+  navigation?: any; // TODO
```

Comment:
seems that it doesn't make sense to merge it with `any` here as turn off checking types completely for navigation. If it needs a refactor to make it work I would make it and merge it after a hard fork

Reply:
yes for me to it is not resolve